package.json

```diff
@@ -38,7 +38,8 @@
     }
   ],
   "dependencies": {
-    "depd": "2.0.0"
+    "depd": "2.0.0",
+    "xml2js": "0.4.23"
```

Comment:
Do we need to be so specific with the version?

Reply:
I think we can safely use `~0.4.23` or `^0.4.23`

![image](https://user-images.githubusercontent.com/33973272/75493875-84fbc580-5980-11ea-9a66-65035a560c17.png)

Reply:
I will update this PR to use `^0.4.23`

Reply:
We should probably update depd to `^2.0.0` in a future PR as well.